dimod/generators/binpacking.py

```diff
@@ -53,15 +54,15 @@ def random_bin_packing(num_items: int,
     bin_capacity = int(num_items * np.mean(weights) / 5)
     model = ConstrainedQuadraticModel()
 
-    obj = BinaryQuadraticModel(vartype='BINARY')
+    obj = BinaryQuadraticModel(BINARY)
     y = {j: obj.add_variable(f'y_{j}') for j in range(max_num_bins)}
 
     for j in range(max_num_bins):
         obj.set_linear(y[j], 1)
 
     model.set_objective(obj)
 
-    x = {(i, j): model.add_variable(f'x_{i}_{j}', vartype='BINARY') for i in range(num_items) for
+    x = {(i, j): model.add_variable(BINARY, f'x_{i}_{j}') for i in range(num_items) for
          j in range(max_num_bins)}
 
```

Comment:
I think you have more uses to replace: for example, https://github.com/dwavesystems/dimod/blob/93796d0ceb21d4d98ef03c10a0b42ff1f88e32a8/dimod/constrained.py#L407

Reply:
This is why I didn't want to ignore results on `add_variable()`! :laughing: